window.go

```diff
@@ -519,3 +521,11 @@ func (w *Window) Unmaximize() (err error) {
 	_, err = synchronousEvent(w.ctx, w, w.w, Event{Name: EventNameWindowCmdUnmaximize, TargetID: w.id}, EventNameWindowEventUnmaximize)
 	return
 }
+
+func (w *Window) ExecuteJavaScript(code string) (err error) {
+	if err = w.ctx.Err(); err != nil {
+		return
+	}
+	_, err = synchronousEvent(w.ctx, w, w.w, Event{Name: EventNameWindowCmdWebContentsExecutedJavaScript, TargetID: w.id, Code: code}, EventNameWebContentsExecuteJavaScript)
```

Comment:
This should be `_, err = synchronousEvent(w.ctx, w, w.w, Event{Name: EventNameWindowCmdWebContentsExecuteJavaScript, TargetID: w.id, Code: code}, EventNameWindowEventWebContentsExecutedJavaScript)`